releasenotes/notes/rpc_package_index-a03d4d131c16d710.yaml

```diff
@@ -0,0 +1,12 @@
+---
+upgrade:
+  - The ``repo_build_pip_extra_index`` should be
+    replaced by a new variable (list type) named
+    ``repo_build_pip_extra_indexes``. This is required
+    if the deployer uses a python package index in addition
+    to the standard PyPI.
+
+deprecations:
+  - The ``repo_build_pip_extra_index`` variable is now
+    deprecated, and the deployer should use the list
+    ``repo_build_pip_extra_indexes`` instead.
```

Comment:
I'm not sure these notes are useful as currently written, they do not feel very specific to RPCO. For a standard RPCO deployment, the requirement to set repo_build_pip_extra_indexes so that rpc-repo is used, is completely internal to the project. I'm not sure we should account for the potential that this var, or some other related var, has been overidden because ultimately support can use anything OSA puts at their disposal. I think there is a reasonable case to say that a release note should be added because the var is being added to user_variables but as it stands the release notes do not seem to address that point.